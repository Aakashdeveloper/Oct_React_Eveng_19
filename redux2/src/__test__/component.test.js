import React from 'react';
import LatestNews from '../component/Home/LatestNews';
import { create} from 'react-test-renderer';

describe('testing Component snapshot', () => {
    test('latestNews', () => {
        let tree = create(<LatestNews/>)
        expect(tree.toJSON()).toMatchSnapshot();
    })
})