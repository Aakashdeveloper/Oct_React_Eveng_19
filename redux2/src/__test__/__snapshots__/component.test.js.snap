// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Component snapshot latestNews 1`] = `
<div
  className="home-latest testapp"
/>
`;
